airflow/dag_processing/processor.py

```diff
@@ -350,7 +350,7 @@ def done(self) -> bool:
         return False
 
     @property
-    def result(self) -> tuple[int, int] | None:
+    def result(self) -> tuple[int, int, int] | None:
```

Comment:
Looks good to me, however, I'm afraid this might be considered as a breaking change if this class is a part of the public API. Isn't it?

Reply:
This `result` function is used only in one protected method and this call does not break any.  But I am sharing your concern.